apps/parse/config/config.exs

```diff
@@ -6,7 +6,7 @@ config :parse, Facility.Parking,
   garages: %{
     "Alewife" => "park-alfcl-garage",
     "Braintree" => "park-brntn-garage",
-    # Woodland
+    "Woodland" => "park-woodl-garage",
```

Comment:
Does it make sense to still leave the `"DataPark" => "park-woodl-garage"` map entry below just in case they ever fall back to using it or something? Or would it be better to just remove it? I suppose we have map entries for both `MBTA SALEM` and `MBTA SALEM ` (with the trailing space)...

Reply:
I thought about that, and figured that it's better to be safer and keep both versions around.